src/YProperty.h

```diff
@@ -150,6 +150,19 @@ class YPropertyValue
     ~YPropertyValue();
 
     /**
+     * Equyality operator, can compare with another YPropertyValue.
+     * @return true if the value is the same, throws an exception for
+     *   incompatible property types.
+     */
+    bool operator==( const YPropertyValue &other ) const;
+
+    /** Inequality operator
+     * TODO:  Maybe it can be added automatically by some boost "magic"...
```

Comment:
So Boost *does* have a library for this: http://www.boost.org/doc/libs/1%5F39%5F0/libs/utility/operators.htm#smpl_oprs but I think it would be overkill to use it for this. Let's remove the TODO.